docs/src/research/installation.md

```diff
@@ -0,0 +1,60 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# System Installation
+
+## Overview
+
+A secure system installation covers multiple phases - from establishing trust to installation process. This document discusses learnings developing mechanisms to setup a Ghaf system in a target hardware.
```

Comment:
Because nothing ever is 100 % secure, unless there is a political requirement to use the word "secure", I would use some other word like "hardened".

Reply:
Then again - Secure System Research Center. Secure is a good target even when securing or hardening is a continuous process always striving for better. I changed the term from "secure" to "hardened", though.